### PR TITLE
mailroom-pt3.py updated for re-grade

### PR DESCRIPTION
**Pull request by nichx313**
updated, including strip of manual tests - pytest passes on all modules

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
@nichx313 can you please clarify what you would like to get re-graded?

lesson06 mailroom has full grade, thanks.

**Comment by nichx313** [comment]
Sorry, lesson05/mailroom-pt3/assignment4 is the one I would like to get a second look. I’ve corrected an index error there that cost me a few points. Thanks!

**Comment by natasha-aleksandrova** [comment]
@nichx313 I don't see any files for lesson05 in this commit?

**Comment by nichx313** [comment]
I’ll push a new change in an hour or so. Sorry.

**Comment by nichx313** [comment]
Should be there now.

File: students/nick_miller/lesson05/mailroom-pt3.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-"""PY210_SP - mailroom-part 3 - done
+"""PY210_SP - mailroom-part 3 - index error fixed
 author: Nick Miller"""
```
Comment (natasha-aleksandrova) [inline students/nick_miller/lesson05/mailroom-pt3.py]: Is there a code change in this file? I just see a comment being updated.

**Comment by nichx313** [comment]
It was fixed several pushes back. The name change was just to get it to the top.

**Comment by natasha-aleksandrova** [comment]
closing since there is nothing to review. grade was adjusted.

**Pull request closed without merging**